### PR TITLE
Update the format of the case study

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #<NUM>

* The case study needed to be updated upon discussion with KW

### List of changes proposed in this PR (pull-request)

* If the structure of the case study looks ok - @pgadige can look at it to help structure the template for case studies
* What information is missing that should be added

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
@pgadige can you please review this keeping in mind how we can best create a template for Case Studies?

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md [hunk 1]
```diff
@@ -186,8 +190,7 @@ docker exec --name mycontainer /
     snakemake -F --cores 1  manuscript
 ```
 
-
-## Long term archiving and citeability
+### Long term archiving and citeability
 
 The GitHub repository is also linked with [zenodo.org]() to ensure long-term
```
Comment (pgadige) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:195]: Add link for zenodo.org - https://zenodo.org/

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md [hunk 2]
```diff
@@ -1,13 +1,33 @@
...
 
+## About this case study
 
-## A short summary of the Manuscript
+The purpose of this case study is to discuss the different components of 
+research reproducibility implemented in designing and conducting a 
+statistical study.
+With the help of their manuscript, the authors provide a catalog of methods
+used in their research and cross-reference them to the respective
+sections discuss in this {ref}`rr`.
```
Comment (pgadige) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:11]: discussed

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md [hunk 3]
```diff
@@ -1,13 +1,33 @@
...
+
+## About of the Manuscript
```
Comment (pgadige) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:13]: About the Manuscript

**Review by pgadige — DISMISSED** [review]
Save a few modifications, the overall format looks good.

**Comment by pgadige** [comment]
> @pgadige can you please review this keeping in mind how we can best create a template for Case Studies?

@malvikasharan I've reviewed the new changes. The format looks good. We may need a few modifications. Thank you.

**Comment by malvikasharan** [comment]
Thanks Pooja. Please have a look at the low-level case studies in this book to get some inspiration on how we would like to have our case studies to look like https://www.practicereproducibleresearch.org/core-chapters/4-casestudies.html

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md (hunk 2 above)
Comment (malvikasharan) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:11]:
```suggestion
sections discussed in this {ref}`rr`.
```

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md (hunk 3 above)
Comment (malvikasharan) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:13]:
```suggestion
## About the Manuscript
```

File: book/website/reproducible-research/case-studies/statistical-methods-manuscript.md (hunk 1 above)
Comment (malvikasharan) [inline book/website/reproducible-research/case-studies/statistical-methods-manuscript.md:195]:
```suggestion
The GitHub repository is also linked with [zenodo.org](https://zenodo.org/) to ensure long-term
```

**Comment by malvikasharan** [comment]
@pgadige whenever you are ready, can you please see if this can be approved? I need to share the format with a contributor writing case studies at the moment.

**Comment by malvikasharan** [comment]
@paulowoicho Can you please have a look at the case study structure in this PR and share your opinion with me?

**Comment by paulowoicho** [comment]
> @paulowoicho Can you please have a look at the case study structure in this PR and share your opinion with me?

Hi @malvikasharan, I think the structure looks good especially because it adequately relates to the key concepts discussed in the Reproducible Research Guide. One addition could be a reflections section where the authors talk about any challenges they may have faced and/or key insights learned.

**Comment by malvikasharan** [comment]
Good point on adding the reflection section in the template - I would though go ahead and merge this by opening a new issue for the template series. If you agree, can you please approve this?

**Review by paulowoicho — APPROVED** [review]

**Pull request merged**